src/ui_ng/src/i18n/lang/en-us-lang.json

```diff
@@ -311,7 +312,6 @@
         "TEST_CONNECTION_SUCCESS": "Connection tested successfully.",
         "TEST_CONNECTION_FAILURE": "Failed to ping endpoint.",
         "NAME": "Name",
-        "STATUS": "Status",
```

Comment:
check status is using or not.

Reply:
This "STATUS" is duplicated with line 344